Dockerfile

```diff
@@ -38,6 +38,10 @@ FROM base
 # these libs are necessary for operation
 RUN apk --no-cache add libmagic cairo pango ttf-opensans && \
     apk --no-cache add postgresql-libs libxslt jpeg zlib libxml2 # those *might* be useful
+RUN apk --no-cache add fontconfig=2.14.2-r3
```

Comment:
Please add the apk call to the previous `RUN`
Dockerfile best practice is to reduce RUN statements to reduce layering. Thanks

Reply:
Hi @norbusan 
I didn't add to the previous RUN because we need to label all version for package or the Docker CI test will failed.
As check, I'm not sure which packages are using with correct version; hence, I have to create the new layer for new change.